README.md

```diff
@@ -43,7 +43,8 @@ Creates a new `Shopify` instance.
   `apiKey` and `password` options. If you are looking for a premade solution to
   obtain an access token, take a look at the [shopify-token][] module.
 - `apiVersion` - Optional - A string to specify the [Shopify API
-  version][api-versioning] to use for requests
+  version][api-versioning] to use for requests (by default it's not specified
+  and it will fallback to the oldest)
```

Comment:
Please use

"... to use for requests. Defaults to the oldest supported stable version."